### PR TITLE
add fix for STATIC_ROUTE_EXPIRY_TIME key not exist

### PR DESCRIPTION
**Pull request by jcaiMR**
#### Why I did it
when key STATIC_ROUTE_EXPIRY_TIME is not exist in APPL_DB, self.db.get(self.db.APPL_DB, "STATIC_ROUTE_EXPIRY_TIME", "time") will rise runtime error, and python thread silent exist.

#### How I did it
Add key check before access it's value

#### How to verify it
Try it on DUT

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: src/sonic-bgpcfgd/bgpcfgd/static_rt_timer.py [hunk 1]
```diff
@@ -16,6 +16,9 @@ def __init__(self):
 
     def set_timer(self):
         """ Check for custom route expiry time in STATIC_ROUTE:EXPIRY_TIME """
+        keys = self.db.keys(self.db.APPL_DB, "STATIC_ROUTE_EXPIRY_TIME")
+        if len(keys) == 0:
```
Comment (StormLiangMS) [inline src/sonic-bgpcfgd/bgpcfgd/static_rt_timer.py:20]: Should we add a log here to prove this thread is still alive? Maybe info/debug level?

**Review by StormLiangMS — APPROVED** [review]
Approved with minor suggestion.

File: src/sonic-bgpcfgd/bgpcfgd/static_rt_timer.py (hunk 1 above)
Comment (jcaiMR) [reply src/sonic-bgpcfgd/bgpcfgd/static_rt_timer.py:20]: When key not exists. there will be log periodicly printed like :
 WARNING bgp#bgpcfgd: :- operator(): DB '{APPL_DB}' is empty with pattern 'STATIC_ROUTE_EXPIRY_TIME'!



When key exist there will be log periodicly printed  like:
INFO bgp#bgpcfgd: Static route expiry set to 160s

**Comment by qiluo-msft** [comment]
@liuh-80 to check whether this is swss-common bug. If yes, we need to fix in swss-common and revert this.

**Comment by liuh-80** [comment]
Ack, I will check if this is a swsscommon issue.

**Comment by liuh-80** [comment]
Can'r reproduce this issue with following code, will discussion offline:

>>> from swsscommon import swsscommon
>>> db = swsscommon.SonicV2Connector()
>>> db.connect(db.APPL_DB)
>>> db.keys(db.APPL_DB, "STATIC_ROUTE_EXPIRY_TIME")
[]
>>> db.get(db.APPL_DB, "STATIC_ROUTE_EXPIRY_TIME", "time")
>>>

**Pull request merged**